`witnet_core` build fails when inside the project directory

If I move inside `witnet-rust/core` directory and execute `cargo build` the following error happens:
```
   Compiling witnet_core v0.1.0 (/home/anler/src/public/witnet-rust/core)
error[E0432]: unresolved import `witnet_storage::backends::rocks`
 --> core/src/actors/storage_manager.rs:3:31
  |
3 | use witnet_storage::backends::rocks::RocksStorage;
  |                               ^^^^^ Could not find `rocks` in `backends`

warning: unused import: `witnet_storage::storage::Storage`
 --> core/src/actors/storage_manager.rs:5:5
  |
5 | use witnet_storage::storage::Storage;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(unused_imports)] on by default

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: Could not compile `witnet_core`.

To learn more, run the command again with --verbose.
```

--- a/core/Cargo.toml
+++ b/core/Cargo.toml
@@ -13,6 +13,6 @@ futures = "0.1"
 tokio = "0.1.11"
 bytes = "0.4"
 rand = "0.5.5"
-witnet_storage = { path = "../storage" }
+witnet_storage = { path = "../storage", features = ["rocksdb-backend"] }
 witnet_p2p = { path = "../p2p" }
 witnet_util = { path = "../util" }

